src/pandascore_sync.py

```
@@ -209,6 +232,14 @@ async def perform_pandascore_sync(
     )
 
     async with get_async_session() as db_session:
+        # Attach desired game to the session object so lower-level helpers
+        # can pick it up when needed (lightweight pragmatic approach).
+        try:
+            setattr(db_session, "_sync_game", game or None)
+        except Exception:
+            # Non-fatal: if we cannot attach, parser selection falls back
+            # to defaults in _process_matches_and_commit.
+            pass
```

Ruff (0.15.5)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)